6-AdvancedScenarios/3-call-api-acrs/AppCreationScripts/Configure.ps1

```diff
@@ -193,194 +171,183 @@ Function ConfigureApplications
-   Write-Host -ForegroundColor Green "------------------------------------------------------------------------------------------------" 
-   Write-Host "IMPORTANT: Please follow the instructions below to complete a few manual step(s) in the Azure portal":
-   Write-Host "- For service"
-   Write-Host "  - Navigate to Azure portal and set the 'accessTokenAcceptedVersion' to '2' in the application manifest" -ForegroundColor Red
-   Write-Host "  - Navigate to the API Permissions page and select 'Grant admin consent for (your tenant)" -ForegroundColor Red
-   Write-Host "  - Navigate to the Manifest page, find the 'optionalClaims' section and change its default value to request 'xms_cc' claims" -ForegroundColor Red  
-   Write-Host "- For client"
-   Write-Host "  - Navigate to Azure portal and set the 'replyUrlsWithType' to 'Spa' in the application manifest" -ForegroundColor Red 
-
-   Write-Host -ForegroundColor Green "------------------------------------------------------------------------------------------------" 
-      if($isOpenSSL -eq 'Y')
-   {
+    Set-AzureADApplication -ObjectId $serviceAadApplication.ObjectId -KnownClientApplications $knowApplications
+    Write-Host "Configured."
+
+    # Update config file for 'service'
+    $configFile = $pwd.Path + "\..\API\.env"
+    Write-Host "Updating the sample code ($configFile)"
+    $dictionary = @{ "Enter_the_Application_Id_Here" = $serviceAadApplication.AppId; "Enter_the_Tenant_Info_Here" = $tenantId; "Enter_the_Client_Secret_Here" = $serviceAppKey };
+    ReplaceInTextFile -configFilePath $configFile -dictionary $dictionary
+
+    # Update config file for 'client'
+    $configFile = $pwd.Path + "\..\SPA\src\authConfig.js"
+    Write-Host "Updating the sample code ($configFile)"
+    $dictionary = @{ "Enter_the_Application_Id_Here" = $clientAadApplication.AppId; "Enter_the_Tenant_Info_Here" = $tenantId; "Enter_the_Web_Api_Scope_here" = ("api://" + $serviceAadApplication.AppId + "/access_as_user") };
+    ReplaceInTextFile -configFilePath $configFile -dictionary $dictionary
+    Write-Host -ForegroundColor Green "------------------------------------------------------------------------------------------------" 
+    Write-Host "IMPORTANT: Please follow the instructions below to complete a few manual step(s) in the Azure portal":
+    Write-Host "- For service"
+    Write-Host "  - Navigate to Azure portal and set the 'accessTokenAcceptedVersion' to '2' in the application manifest" -ForegroundColor Red
+    Write-Host "  - Navigate to the API Permissions page and select 'Grant admin consent for (your tenant)" -ForegroundColor Red
+    Write-Host "  - Navigate to the Manifest page, find the 'optionalClaims' section and change its default value to request 'xms_cc' claims" -ForegroundColor Red  
+    Write-Host "- For client"
+    Write-Host "  - Navigate to Azure portal and set the 'replyUrlsWithType' to 'Spa' in the application manifest" -ForegroundColor Red 
```

Comment:
Since the code is stable now, lets use the new code gen to get rid of as many manual steps as possible